doc(vitals-prometheus) add guide to relabel instance in prometheus (#1550)
Kong/docs.konghq.com: app/enterprise/0.35-x/admin-api/vitals/vitals-prometheus-strategy.md

[scape_configs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#%3Cscrape_config%3E)
section of Prometheus document for further reading.

> By default, Vitals graph in Kong Manager uses the configured target address

Reviewer: ```suggestion
By default, the Vitals graph in Kong Manager uses the configured target address
```

> in legend, which is named `instance` in Prometheus metrics label. For some service

Reviewer: ```suggestion
in the legend, which is named `instance` in the Prometheus metrics label. For some service
```

> discovery setup where `instance` is IP:PORT, user might want to relabel `instance`

Reviewer: ```suggestion
discovery setups where `instance` is `IP:PORT`, the user might want to relabel the `instance`
```

> label to display more meaningful hostname in legend.

Reviewer: ```suggestion
label to display more a meaningful hostname in the legend.
```

> To do so, user can also refer to [scape_configs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#%3Cscrape_config%3E)

Reviewer: ```suggestion
To do so, the user can also refer to the [scape_configs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#%3Cscrape_config%3E)
```

> section and rewrite `instance` label with corresponding meta label.

Reviewer: ```suggestion
section and rewrite the `instance` label with the corresponding meta label.
```
